crowbar_framework/app/assets/stylesheets/content/content.scss

```diff
@@ -45,21 +52,21 @@
   }
 }
 
-.dropdown-menu {  
-  border-radius: 6px;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  right: -1px;
+h1, h2, h3.modal-title {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line